src/ensembl/src/content/home/Home.scss

```diff
@@ -12,7 +12,8 @@
 
 .search {
   background: $ens-light-grey;
-  box-shadow: 0 3px 10px $ens-grey;
+  box-shadow: 0 3px 5px $ens-medium-light-grey;
```

Comment:
Does this mean that the `.speciesSearchPanel` class (see SpeciesSelector screen) should have the same box shadow (it currently has none)?

Reply:
🤷‍♂ I guess this concern was extracted into ENSWBSITES-291

Reply:
Yes